flutter_local_notifications/lib/src/platform_specifics/linux/notification_details.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:meta/meta.dart';
+
+import 'icon.dart';
+
+/// Signature of callback which triggered when user taps 
+/// a button of a notification.
+typedef LinuxNotificationButtonHandler = void Function(int id, String buttonId);
+
+/// Details of notification button on Linux.
+@immutable
+class LinuxNotificationButton {
+  /// Construct an instance of [LinuxNotificationButton].
+  const LinuxNotificationButton(
+    {@required this.label, @required this.buttonId, @required this.handler});
+
+  /// Label of this button.
+  final String label;
+  /// Identifier of this button, will be passed to callback.
```

Comment:
Can you run `flutter format .` from the root of the plugin? Regular formatting would add empty lines between properties and running the code does seem to show that some files would be changes as a result. Note that if happen to be using VS Code that you can turn format on save on, which is an option that is off by default